src/current/v23.2/live-migration-service.md

```diff
@@ -24,51 +24,64 @@ This page describes how to [install](#installation), [configure](#configuration)
 
 - A *live migration* keeps two production databases online (a source and a target database) and uses either replication or dual writing to keep data identical between them until a final cutover.
 - The *source of truth* is the database that serves reads and writes to the application during a live migration. A cutover switches the source of truth.
-- *Shadowing* is the replaying of traffic on the target database. The LMS supports multiple [shadowing modes](#shadowing-modes).
+- *Shadowing* is the replaying of source traffic on the target database. The LMS supports multiple [shadowing modes](#shadowing-modes).
 
 ## Requirements
 
 - [Kubernetes](https://kubernetes.io/) cluster
 - [Helm](https://helm.sh/docs/intro/install/) package manager for Kubernetes
 
-#### Supported databases
+#### Supported database technologies
 
 - [PostgreSQL]({% link {{ page.version.version }}/migrate-from-postgres.md %}) (source)
 - [MySQL]({% link {{ page.version.version }}/migrate-from-mysql.md %}) (source)
 - CockroachDB (source and target)
 
 ## Installation
 
-Add the [Helm chart repository](https://molt.cockroachdb.com/lms/charts/) with [`helm repo add`](https://helm.sh/docs/helm/helm_repo_add/). Then install the chart with [`helm install`](https://helm.sh/docs/helm/helm_install/).
+1. Add the [Helm chart repository](https://molt.cockroachdb.com/lms/charts/) with [`helm repo add`](https://helm.sh/docs/helm/helm_repo_add/). Then install the chart with [`helm install`](https://helm.sh/docs/helm/helm_install/).
 
-To set up the LMS resources, [install `molt-lms-cli`](#molt-lms-cli) and run the following command:
+1. Port-forward from your local machine to the orchestrator:
 
-{% include_cached copy-clipboard.html %}
-~~~ shell
-molt-lms-cli initialize
-~~~
+	{% include_cached copy-clipboard.html %}
+	~~~ shell
+	kubectl port-forward svc/lms-molt-lms-orchestrator 4200:4200 
```

Comment:
Since we dropped the `molt` part of the chart name, the orchestrator is just simply `lms-orchestrator` now. And same principal, if your releasename is lms, then the leading lms is dropped and you just have `lms-orchestrator`

Reply:
Fixed.